### ISSUE TITLE
[ACCESSIBILITY] Button is in improper sequence and incorrectly announced - Important issue 

### ISSUE THREAD

**Issue by timo54** [rokwire/illinois-app#71]
**Describe the bug**
When swiping through the Color Definition popup window, the swipe order takes the user through the initial text and then directly to the Close button. In stead, it should go through ALL the text as shown in the screen capture and then to the Close button. Blind users will assume there's no more text when they encounter the close button. Also, the close button on the Status Card Color Definition window is announced as "Multiplication sign, double-tap to activate". This is incorrect and not an accurate description. 

**To Reproduce**
1. Open sceen reading software
2. Swipe to the Information window on the Status Card
3. Swipe to through the text and then to the close button on the Status Card Color Definition window
4. See the incorrect Swipe order and hear the screen reader incorrectly identify the button.

**Expected behavior**
Swipe order should be logical and go through the text before reaching the close button. Also, buttons and links need to be properly labeled.

**Screenshots**
See below.

**Smartphone (please complete the following information):**
 - Device: Google Pixel
 - OS: Android
 - Version 2.5.8

<img width="420" alt="MultiplicationClose Path" src="https://user-images.githubusercontent.com/12497650/92806428-9e45eb80-f37f-11ea-9a4b-2b075ed87d43.png">

**Comment by timo54** [comment]
This issue has been closed here and moved to the Safer Illinois repository.
